aws/deregister_amis.py

```diff
@@ -66,18 +66,6 @@ def main(raw_args):
         "'delete' as the tag value."
     )
     parser = argparse.ArgumentParser(description=desc)
-    parser.add_argument(
-        '--aws-access-key-id', '-i',
-        dest='aws_access_key_id',
-        help='aws access key id',
-        default=os.environ.get("AWS_ACCESS_KEY_ID"),
-        )
-    parser.add_argument(
-        '--aws-secret-access-key', '-s',
-        dest='aws_secret_access_key',
-        help='aws secret access key',
-        default=os.environ.get("AWS_SECRET_ACCESS_KEY"),
-    )
```

Comment:
^ won't we still need these? Or is the fact that the jenkins worker has an IAM role what allows this script to function w/o credentials?

Reply:
They're picked up from the environment variables.  boto3 dropped easy support for passing these in explicitly via the API in order to discourage people from listing the credentials in their source code.  And in practice, our Jenkins job wasn't using these command line parameters.